auth/consts_test.go

```diff
@@ -0,0 +1,78 @@
+package auth
+
+import (
+	pbauth "github.com/hwsc-org/hwsc-api-blocks/lib"
+	"time"
+)
+
+const (
+	structHeader = iota
+	structBody
+)
+
+var (
+	validCreatedTimestamp    = time.Date(2019, 1, 1, 0, 0, 0, 0, time.UTC).Unix()
+	validExpirationTimestamp = time.Unix(validCreatedTimestamp, 0).AddDate(30, 0, 0).UTC().Unix()
+	validSecretKey           = "j2Yzh-VcIm-lYUzBuqt8TVPeUHNYB5MP1gWvz3Bolow="
+	// valid256JWT and encoded256JWTHeader are dependent to each other
+	valid256JWT = &Header{
+		Alg:      Hs256,
+		TokenTyp: Jwt,
+	}
+	encoded256JWTHeader = "eyJBbGciOjEsIlRva2VuVHlwIjoxfQ"
+	// valid512JWT and encoded512JWTHeader are dependent to each other
+	valid512JWT = &Header{
+		Alg:      Hs512,
+		TokenTyp: Jwt,
+	}
+	encoded512JWTHeader = "eyJBbGciOjIsIlRva2VuVHlwIjoxfQ"
+	// validAdminBody and encodedAdminBody are dependent to each other
+	validAdminBody = &Body{
+		UUID:                "01d3x3wm2nnrdfzp0tka2vw9dx",
+		Permission:          Admin,
+		ExpirationTimestamp: time.Date(2030, 1, 1, 0, 0, 0, 0, time.UTC).Unix(),
+	}
+	encodedAdminBody = "eyJVVUlEIjoiMDFkM3gzd20ybm5yZGZ6cDB0a2Eydnc5ZHgiLCJQZXJtaXNzaW9uIjozLCJFeHBpcmF0aW9uVGltZXN0YW1wIjoxODkzNDU2MDAwfQ"
+	// validUserBody and encodedUserBody are dependent to each other
+	validUserBody = &Body{
+		UUID:                "22d3x3wm2nnrdfzp0tka2vw9dx",
+		Permission:          User,
+		ExpirationTimestamp: time.Date(2030, 1, 1, 0, 0, 0, 0, time.UTC).Unix(),
+	}
+	encodedUserBody = "eyJVVUlEIjoiMjJkM3gzd20ybm5yZGZ6cDB0a2Eydnc5ZHgiLCJQZXJtaXNzaW9uIjoyLCJFeHBpcmF0aW9uVGltZXN0YW1wIjoxODkzNDU2MDAwfQ"
+	validSecret     = &pbauth.Secret{
+		Key:                 validSecretKey,
+		CreatedTimestamp:    validCreatedTimestamp,
+		ExpirationTimestamp: validExpirationTimestamp,
+	}
+	valid512JWTAdminTokenString = "eyJBbGciOjIsIlRva2VuVHlwIjoxfQ.eyJVVUlEIjoiMDFkM3gzd20ybm5yZGZ6cDB0a2Eydnc5ZHgiLCJQZXJtaXNzaW9uIjozLCJFeHBpcmF0aW9uVGltZXN0YW1wIjoxODkzNDU2MDAwfQ.8lVhZo_W6KmGI2oi5JNHioDvPq2Yl86v4uae3RfKc-qoKUwHNxFtXO2NFmChsi35__t1uC_SD-Ay_MoateeDNg=="
```

Comment:
can we break these into multiple lines so it is not going over ide's set column line